README.md

```diff
@@ -1 +1,80 @@
-# JS-Capstone
\ No newline at end of file
+# JavaScript Group Capstone Project
+
+> This project is about meal recepies, categorically organizied by alphabetic order. the user accesses the site to checkout a recepie for a meal and likes the meal or even leaves a comment about the meal after trying it out
+
+![screenshot]()

```

Comment:
- [ ] <b>REQUIRED:</b> The screenshot is showing no image because it is empty. kindly fix it.